mail-storage/src/main/java/mail/storage/exception/DraftMessageException.java

```diff
@@ -0,0 +1,10 @@
+package mail.storage.exception;
+
+import lombok.experimental.StandardException;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.BAD_REQUEST)
```

Comment:
I would handle it in contoller advuce and this annotation is experimental

Reply:
Fixed.